### PR TITLE
[HH][105695446] throws an event to hook into for tagging

### PR DESCRIPTION
**Pull request by HansHauge**
https://www.pivotaltracker.com/story/show/105695446

Just throws an event we can listen for in order to tag the carousel images.

[Rentals PR](https://github.com/rentpath/rentals/pull/1005)

### CODE REVIEW

File: ui/carousel.js [hunk 1]
```diff
@@ -72,6 +72,7 @@
         });
         scrollable.smoothDivScroll("recalculateScrollableArea");
         scrollable.smoothDivScroll("startAutoScrolling");
+        $(document).trigger('uiTagCarouselImages');
```
Comment (mpelzsherman) [inline ui/carousel.js]: We should make this change to carousel.coffee as well.

I'd suggest something more specific the the event you are tagging. `uiTagCarouselImages` is a bit generic. 

The story is unclear about which events we're tagging, BTW.

**Comment by mpelzsherman** [comment]
@michaelstalker - we should commit both the coffeescript and generated .js files for this project, right?

**Comment by michaelstalker** [comment]
@mpelzsherman Let's talk about this tomorrow, or on Monday if I'm not well enough to work tomorrow.

**Comment by michaelstalker** [comment]
I'm still getting requirements for the story this PR addresses. This code change might not be necessary. I'm closing this for now. We may reopen it once we hear back from the tagging team.

**Pull request closed without merging**